src/main/example/src/game.cpp

```diff
@@ -241,14 +256,15 @@ int mainLoop(gameInfo* gamein) {
         if (error) {
             return error;
         }
-        collision = currentGame->getCollision(playerRef, wolfRef, vec3(0, 0, 0));
+        auto newCollision = currentGame->getCollision(playerRef, wolfRef, vec3(0, 0, 0));
         string collMessage;
-        if (collision == 1) {
+        if (newCollision == 1) {
             collMessage = "Contact: True";
         } else {
             collMessage = "Contact: False";
         }
-        collDebugText->setMessage(collMessage);
+        if (newCollision != collision) collDebugText->setMessage(collMessage);
```

Comment:
Added this extra logic into the mainLoop to prevent the collision message from changing every frame... The text only needs to change when the message is actually changed.

Reply:
Nevermind. Moved this functionality inside of the setMessage function.